app/src/userpages/modules/transactionHistory/services.js

```diff
@@ -138,25 +138,31 @@ export const getTransactionsFromEvents = (events: EventLogList): Promise<Transac
         web3.eth.getTransactionReceipt(event.transactionHash),
         web3.eth.getBlock(event.blockHash),
     ])))
-        .then(([...transactions]) =>
-            transactions.map(([event, tx, receipt, block]): TransactionEntity => {
-                const rest = {}
-
-                if (receipt.status === true) {
-                    rest.receipt = receipt
-                } else {
-                    rest.error = new TransactionError(I18n.t('error.txFailed'), receipt)
-                }
+        .then(([...transactions]) => transactions.map(([event, tx, receipt, block]): TransactionEntity => {
+            const rest = {}
 
-                return {
-                    id: event.id,
-                    hash: tx.hash,
-                    state: 'completed',
-                    gasUsed: receipt.gasUsed,
-                    gasPrice: tx.gas,
-                    timestamp: block.timestamp,
-                    ...getInputValues(event.type, receipt.logs),
-                    ...rest,
-                }
-            }))
+            if (receipt.status === true) {
+                rest.receipt = receipt
+            } else {
+                rest.error = new TransactionError(I18n.t('error.txFailed'), receipt)
+            }
+
+            let inputValues
+
+            try {
+                inputValues = getInputValues(event.type, receipt.logs)
```

Comment:
only fix here is to wrap `getInputValues` in `try..catch` so that the whole list doesn't die if one call fails